ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -54,9 +54,13 @@ namespace ngraph
                             splits[0].get_node_shared_ptr(),
                             num_groups_const,
                             std::make_shared<default_opset::Divide>(splits[1], num_groups_const)};
+
+                        auto zero_const =
+                            default_opset::Constant::create(element::i64, Shape{1}, {0});
                         for (size_t i = 2; i < rank_size; i++)
                         {
-                            new_shape.push_back(splits[i].get_node_shared_ptr());
+                            new_shape.push_back(
+                                std::make_shared<default_opset::Add>(splits[i], zero_const));
```

Comment:
I'm trying to understand - how does adding a zero fix the shape from `{1, 8, 2, 1, 1, 1}` to `{1, 8, 2, 128, 128, 128}`?

Reply:
We had discussed it offline, @tomdol found a PR that has just been merged and is probably fixing this problem:
https://github.com/openvinotoolkit/openvino/pull/3953

Reply:
I have verified behavior after mentioned PR - this PR fixes the problem on the interpreter, but it still occurs on the CPU.

Reply:
Ticket: 50650

Reply:
Please add a comment why this is needed.
Add a new ticket to remove this workaround when the issue with split will be fixed, please.

Reply:
I'm not sure it is really the [`Split`](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Split_1.md) op issue,  expected behaviour here seems rather like [`Gather`](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Gather_1.md)

Reply:
I'm not sure it is really the [`Split`](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Split_1.md) op issue,  expected behaviour here seems rather like [`Gather`](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Gather_1.md)

Reply:
Or `new_shape` should be `OutputVector` instead of `NodeVector`